spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -188,6 +199,23 @@ data class SpdxCompoundExpression(
         right.validate(strictness)
     }
 
+    override fun validChoicesDnf(): Set<SpdxExpression> {
+        return when (operator) {
+            SpdxOperator.AND -> setOf(this)
+            SpdxOperator.OR -> {
+                val validChoicesLeft = when (left) {
+                    is SpdxCompoundExpression -> left.validChoicesDnf()
+                    else -> left.validChoices()
+                }
+                val validChoicesRight = when (right) {
+                    is SpdxCompoundExpression -> right.validChoicesDnf()
+                    else -> right.validChoices()
+                }
+                (validChoicesLeft + validChoicesRight)
```

Comment:
Nit: Blank line before this one. Parentheses could be dropped.

Reply:
Done.